config/application.rb

```diff
@@ -8,6 +8,8 @@
 # require "active_storage/engine"
 require "action_controller/railtie"
 require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
```

Comment:
The why particular ones aren't used? No. Basically instead of loading all Rails frameworks, we only load those that we use. We don't use neither `active_storage` (we use CarrierWave to upload images), nor `action_mailbox` (we don't get inbound emails from the Rails app), nor `action_text` (our editors are custom), nor `action_cable` (no WebSockets for now), or `test_unit` (we use RSpec).